src/Router.js

```diff
@@ -0,0 +1,52 @@
+import React, { lazy } from 'react'
+import { createBrowserHistory } from 'history'
+import { Navigate, Route, Routes } from 'react-router-dom'
+import { nanoid } from 'nanoid'
```

Comment:
it's better to have waterfall rule applied:
E.g: 
```suggestion
import { nanoid } from 'nanoid'
import { createBrowserHistory } from 'history'
import { Navigate, Route, Routes } from 'react-router-dom'
```

excepts the line import from `react` should be placed on top of other imports.

Reply:
please also check other files and apply waterfall rule too.